### PR TITLE
Event toggling snippet

### PR DESCRIPTION
**Pull request by AlexJerabek**
This sample shows how to turn on and off events.

My intention is to make this part of a larger sample once calculation toggling goes online. Hence why there's more code and data here than absolutely necessary.

### CODE REVIEW

File: samples/excel/89-preview-apis/events-disable-events.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to turn on and off events.</p>
```
Comment (ElizabethSamuel-MSFT) [inline samples/excel/89-preview-apis/events-disable-events.yaml]: maybe "turn events on and off" instead

File: samples/excel/89-preview-apis/events-disable-events.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                // Log the address of the sum that changed
+                const range = context.workbook.bindings.getItem(eventArgs.binding.id).getRange();
+                const currentDate = new Date();
+                const cell = range.getCell(1, 0);
+                cell.formulas = [["=TIME(" + currentDate.getHours() + "," + currentDate.getMinutes() + "," + currentDate.getSeconds() + ")"]];
+                await context.sync();
+            });
+        }
+
+        async function setup() {
```
Comment (ElizabethSamuel-MSFT) [inline samples/excel/89-preview-apis/events-disable-events.yaml]: I recommend that you have setup create a new sheet with some appropriate data. Then you don't have to worry about interfering with the user's sheets and the other methods can look for that sheet.

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT Thanks for the feedback. Updated accordingly.

I also changed what the event handler does to be more obvious when it's on or off.

File: samples/excel/89-preview-apis/events-disable-events.yaml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            });
+        }
+
+        async function onSumChanged(eventArgs: Excel.BindingDataChangedEventArgs) {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const cell = sheet.getRange("P21");
+                 
+                // get the grand total of the whole sum range
+                // note that we are using having this program perform the sum instead of giving the Excel cell a function,
```
Comment (ElizabethSamuel-MSFT) [inline samples/excel/89-preview-apis/events-disable-events.yaml]: Should this start with: // note that we are having

**Pull request merged**